### ISSUE TITLE
About training settings

### ISSUE THREAD

**Issue by xtxherry** [cecret3350/DEA-Net#6]
Hello, I am honored to read your article. I would like to ask how much GPU memory is occupied when you use RTS3080ti for training. Also, your entire network has been trained for 1500 iterations, where does it converge?

**Comment by cecret3350** [comment]
Hi @xtxherry , sorry for late reply.
1. I'm sorry I've forgotten the GPU memory used.
2. Our model basically converges around 1300k iterations (probably due to the reduced learning rate).
Hope my answer can help you!

**Comment by xtxherry** [comment]
Thank you for your reply and look forward to your follow-up work.

> Hi @xtxherry , sorry for the late reply.
> 
> 1. I'm sorry I've forgotten the GPU memory used.
> 2. Our model basically converges around 1300k iterations (probably due to the reduced learning rate).
>    Hope my answer can help you!
